### PR TITLE
verbose changes for cost estimate plan popup

### PR DESCRIPTION
**Pull request by Rohithone**
# PR Checklist

Please check if your PR fulfills the following requirements
- [ ] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [ ] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

# What is the ticket # detailing the issue?

Ticket:  https://redmine.priv.dchbx.org/issues/101871

# A brief description of the changes

Current behavior:
After an IVL consumer or EE clicks on Estimate Your Cost in plan shopping to get to the plan match tool, they get a pop-up window with buttons for Cancel or ARE YOU SURE. ARE YOU SURE is not an action.

New behavior:
'Are you sure?' should be added to the end of the paragraph before the buttons and the text on button on the right should be 'Proceed' instead of ARE YOU SURE.

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

### CODE REVIEW

**Comment by polographer** [comment]
please review https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit and update the title of the PR

**Review by polographer — CHANGES_REQUESTED** [review]

File: app/views/insured/plan_shoppings/_plan_filters.html.erb [hunk 1]
```diff
@@ -78,7 +78,7 @@
             <div class="modal-footer" style="padding:10px">
               <center>
               <button type="button" class="btn btn-default btn-default" data-dismiss="modal">Cancel</button>
-              <%=link_to l10n("confirm") , 'https://dc.checkbookhealth.org/dc/', class: "btn btn-primary  mtz", target: '_blank'  %>
+              <%=link_to l10n("proceed") , 'https://dc.checkbookhealth.org/dc/', class: "btn btn-primary  mtz", target: '_blank'  %>
```
[the remark this answers is not in the document]
Comment (Rohithone) [reply app/views/insured/plan_shoppings/_plan_filters.html.erb]: updated the hardcoded url

**Review by utkarsh7989 — DISMISSED** [review]
LGTM

**Comment by Rohithone** [comment]
> please review https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit and update the title of the PR

Done.

**Comment by Rohithone** [comment]
Changes updated.

File: app/views/insured/plan_shoppings/_plan_filters.html.erb [hunk 2]
```diff
@@ -53,12 +53,12 @@
             <div class="modal-footer" style="padding:10px">
               <center>
               <button type="button" class="btn btn-default btn-default" data-dismiss="modal">Cancel</button>
-              <%=link_to l10n("confirm") , 'https://dc.checkbookhealth.org/sadpi/dc/2019/', class: "btn btn-primary" %>
+              <%= sanitize link_to sanitize(l10n("proceed")) , sanitize(l10n("checkbook_sadpi_health_url")), class: "btn btn-primary" %>
```
Comment (polographer) [inline app/views/insured/plan_shoppings/_plan_filters.html.erb:56]: why this line doesn't have only one sanitize like line 35 ?

Comment (Rohithone) [reply app/views/insured/plan_shoppings/_plan_filters.html.erb]: I have updated the code.

**Review by polographer — APPROVED** [review]

**Review by utkarsh7989 — APPROVED** [review]
LGTM

**Pull request merged**